.github/workflows/build.yml

```diff
@@ -0,0 +1,169 @@
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: v1-m2build-${{ hashFiles('app.jar') }}
+          restore-keys: |
+            v1-m2build-
+      - uses: actions/download-artifact@v1
+        with:
+          name: melosys-kodeverk
+          path: melosys-kodeverk-java/melosys-kodeverk/dist
+      - name: Generere kildekode for Java-kodeverk
+        working-directory: melosys-kodeverk-java
+        run: |
+          set -e
+          java -jar ../app.jar melosys-kodeverk/dist/kodemap.yml
+      - name: Bygge Java-kodeverk
+        working-directory: melosys-kodeverk-java/melosys-internt-kodeverk
+        run: mvn clean install -B -V
+      - uses: actions/upload-artifact@v1
+        with:
+          name: melosys-kodeverk-java
+          path: melosys-kodeverk-java
+
+  tagge_frontend:
+    if: github.ref == 'refs/heads/master'
+    needs: bygge_backend
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - run: |
+          # FIXME: Bytt ut med en action for å tagge og committe
+          git config --global user.email "githubci-melosys@nav.no"
+          git config --global user.name "GitHubCI"
+          git remote set-url origin "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}"
+
+          BRANCH=${GITHUB_REF#refs/heads/}
+          git checkout -b ${BRANCH}
+          git branch --set-upstream-to=origin/${BRANCH}
+
+          echo "Tagger ny patch-versjon"
+
+          GIT_COMMIT_DESC=$(git log --format=oneline -n 1 ${GITHUB_SHA})
+          npm version patch -m "%s $GIT_COMMIT_DESC [skip ci]"
+          git push && git push --tags
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  lag_versjon:
+    needs: bygge_frontend
+    runs-on: ubuntu-latest
+    outputs:
+      frontend_versjon: ${{ steps.versjoner.outputs.frontend }}
+      backend_versjon: ${{ steps.versjoner.outputs.backend }}
+    steps:
+      - uses: actions/download-artifact@v1
+        with:
+          name: melosys-kodeverk
+          path: dist
+      - id: versjoner
+        run: |
+          if [ ${IS_MASTER} == true ]; then
+            echo "::set-output name=frontend::$(cat dist/semver.txt)"
+            echo "::set-output name=backend::$(cat dist/semver.txt)"
+          else
+            # GPR tillater ikke samme versjon av ei npm-pakke, selv med --allow-same-versionn
+            echo "::set-output name=frontend::$(cat dist/semver.txt)-beta.${SHORT_SHA}"
+            # GPR git '400 Bad Request' ved bruk av samme versjon
+            echo "::set-output name=backend::$(cat dist/semver.txt)-${SHORT_SHA}-SNAPSHOT"
+          fi
+        env:
+          IS_MASTER: ${{ github.ref == 'refs/heads/master' }}
+          PR_NUMBER: ${{ github.event.number }}
+          SHORT_SHA: ${{needs.bygge_frontend.outputs.short_sha}}
+
+  release_frontend:
+    needs: lag_versjon
```

Comment:
Kom på nå at dette fort kan gi race condition-problematikk for release på master, hvis `tagge_frontend` ikke er ferdig (avhenger av begge byggejobbene, siden de bør være OK for å tagge), og det da vil bli forsøkt å publisere release-artifakt på gammel versjon.

Kunne ha trengt noe à la:
`needs: [lag_versjon, tagge_frontend if github.ref == 'refs/heads/master']
men tipper det må bli separate jobber for snapshot og release.

Reply:
Kjører jobbene i serie fram til release-jobbene.